### ISSUE TITLE
errors with react-intl

### ISSUE THREAD

**Issue by SPAHI4** [iam4x/isomorphic-flux-boilerplate#226]
```
FormattedNumber.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.
```
i just import FormattedNumber with IntlMixin and use it
```
<FormattedNumber  value={ number }/>
```

**Comment by jaredkotoff** [comment]
What version of react-intl do you have installed? I got this when I tried to use 1.2.2. 
You also shouldnt need to import IntlMixin anymore. All you should need is:
```JSX
import { FormattedNumber } form 'react-intl';
```

**Comment by iam4x** [comment]
@SPAHI4 Be to have react-intl version fixed to 1.2.0

https://github.com/iam4x/isomorphic-flux-boilerplate/blob/master/package.json#L66
https://github.com/yahoo/react-intl/issues/241#issuecomment-162875311
